reference-architecture/vmware-ansible/roles/instance-groups/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I'd probably argue that using an included task file that can be referenced by other playbooks may fit this a bit better than a role. Roles really should be agnostic of hosts/groups.